whelktool/scripts/2018/11/fix-musicMedium.groovy

```diff
@@ -0,0 +1,61 @@
+MUSICMEDIUM_TYPE = 'MusicMedium'
+MUSICMEDIUM_PROPERTY = 'musicMedium'
+
+
+boolean remodelToStructuredValue(term) {
+    def remodelledTermList = []
+
+    //Create musicMedium object and add to list
+    term[MUSICMEDIUM_PROPERTY].each {
+        def map = [:]
+        map << ['@type':MUSICMEDIUM_TYPE]
+        map << ['label':it]
+        remodelledTermList << map
+    }
+    term[MUSICMEDIUM_PROPERTY] = remodelledTermList
```

Comment:
This could be shortened to:
```groovy
    term[MUSICMEDIUM_PROPERTY] = term[MUSICMEDIUM_PROPERTY].collect {
        ['@type': MUSICMEDIUM_TYPE, 'label': it]
    }
```

Reply:
Great. I update the code with your suggestion.